test/src/chrome.dart

```diff
@@ -0,0 +1,228 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:devtools/utils.dart';
+import 'package:path/path.dart' as path;
+import 'package:webkit_inspection_protocol/webkit_inspection_protocol.dart'
+    hide ChromeTab;
+import 'package:webkit_inspection_protocol/webkit_inspection_protocol.dart'
+    as wip show ChromeTab;
+
+class Chrome {
+  factory Chrome.from(String executable) {
+    return FileSystemEntity.isFileSync(executable)
+        ? new Chrome._(executable)
+        : null;
+  }
+
+  Chrome._(this.executable);
+
+  static Chrome locate() {
+    if (Platform.isMacOS) {
+      // /Applications/Google\ Chrome\ Canary.app/
+
+      //const String defaultPath = '/Applications/Google Chrome.app';
+      const String defaultPath = '/Applications/Google Chrome Canary.app';
+      //const String bundlePath = 'Contents/MacOS/Google Chrome';
```

Comment:
I assume you meant to leave it at Chrome not Canary before checkin?

Reply:
Yup! Will update.